blocks/form/form.css

```diff
@@ -548,56 +551,58 @@ main .form .file-wrapper .file-drag-area {
 }
 
 /* grid css */
-main .form form .field-wrapper .field-description{
-  grid-column: span 12;
-}
-
-main .form form .field-wrapper.col-12 {
+@media (width >= 900px) {
```

Comment:
theme. but should be part of product since two column layouts do not work properly